Makefile

```diff
@@ -0,0 +1,15 @@
+init: migrate
+	go build main.go
+
+migrate:
+	cp .env.example .env
+	cd db
+	goose sqlite3 ./big_bang_theory_quotes.db up
+	cd ..
+
+down:
```

Comment:
then here we can use "migrate.down", so at least we can differentiate db commands from non-db commands

Reply:
Okay.

Reply:
```suggestion
migrate.down:
```